test/pjrt/test_dynamic_plugin_tpu.py

```diff
@@ -0,0 +1,34 @@
+import concurrent.futures
+
+from absl.testing import absltest
+import torch_xla.core.xla_model as xm
+import torch_xla.distributed.xla_multiprocessing as xmp
+from torch_xla.experimental import plugins
+import torch_xla.runtime as xr
+from torch_xla._internal import tpu
+
+plugins.register_plugin('TPU', tpu.TpuPlugin())
+plugins.use_dynamic_plugins()
+
+
+class TestDynamicTpuPlugin(absltest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    xr.set_device_type('TPU')
+
+  @staticmethod
+  def _assert_tpus_exist(index=0):
+    del index
+    assert len(xm.get_xla_supported_devices('TPU')) > 0
+
+  def test_dynamic_plugin_api(self):
+    with concurrent.futures.ProcessPoolExecutor(max_workers=1) as executor:
```

Comment:
I guess the difference b/w test_dynamic_plugin_api and test_spawn is that the former test single processing and the latter test the multi-processing?

Reply:
Yeah, I wrote `test_dynamic_plugin_api` before the other. I'll change the name to something like `test_single_process` to be more clear